variables.tf

```diff
@@ -562,3 +562,13 @@ variable "ibm_cloud_http_proxy" {
   description = "IBM Cloud http/squid proxy eg. http://10.166.13.64:3128"
   default     = ""
 }
+
+################################################################
+# CSI Driver configuration variables
+################################################################
+
+variable "csi_driver_config" {
+  type        = bool
+  description = "Enable csi-driver configurations (true/false)"
+  default     = true
```

Comment:
Do we need to enable this by default?

Reply:
Yes discussed this with Manju today morning and we expect it to be enabled by default. cc- @mkumatag